release notes/v0.49.0.md

```diff
@@ -15,7 +15,55 @@ k6 `v0.49.0` is here 🎉! This release includes:
 
 ## New features
 
-_optional intro here_
+### Add `clear` to the `locator` class [browser#1149](https://github.com/grafana/xk6-browser/pull/1149)
+
+The new `clear` method on `locator` will clear text boxes and input fields. This is useful when navigating to a website where the text boxes and input fields already contain a value that needs to be cleared before filling it with a specific value.
+
+<details>
+<summary> Expand to see an example of the new functionality.</summary>
+
+```javascript
+import { check } from 'k6';
+import { browser } from 'k6/x/browser';
```

Comment:
I believe this should reference `experimental` instead.
```suggestion
import { browser } from 'k6/experimental/browser';
```

Reply:
Nice catch! Thanks!

Reply:
It was the first one I checked, but then I forgot to mention 😮‍💨 Luckily, we have Daniel 🥰